### PR TITLE
KOGITO-4969: Jobs panel and process diagram migration

### PR DESCRIPTION
**Pull request by Sara4994**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: ui-packages/packages/management-console-webapp/src/components/pages/ProcessDetailsPage/__mocks__/ProcessDetailsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (pefernan) [inline ui-packages/packages/management-console-webapp/src/components/pages/ProcessDetailsPage/__mocks__/ProcessDetailsPage.tsx:17]: header

File: ui-packages/packages/process-list/src/envelope/components/ProcessListTable/ProcessListTable.tsx [hunk 2]
```diff
@@ -86,15 +91,19 @@ const ProcessListTable: React.FC<ProcessListTableProps & OUIAProps> = ({
           id: processInstance.id,
           parent: [
             <>
-              <div>
+              <a
```
Comment (pefernan) [inline ui-packages/packages/process-list/src/envelope/components/ProcessListTable/ProcessListTable.tsx:94]: why not using a patternfly button with link style? (https://www.patternfly.org/v4/components/button#variations)

**Review by AjayJagan — APPROVED** [review]
LGTM 👍

**Review by pefernan — CHANGES_REQUESTED** [review]
@Sara4994 almost there, two minor things:

- ProcessDetailsQueries.ts (line 42) has an issue when quering the details by proccesId, it always resolves with `Promise.resolve(response.data.ProcessInstances[0]);` . This causes that if the response comes with an empty array the promise isn't rejected and the screen stays with the spinner forever.
- Theres a missing badge on the ProcessDetails page title, it shows the process name + the id, in old mgmt-console the id was inside of a badge. 
a part from that everything else is fine! great job

File: ui-packages/packages/process-details/src/envelope/components/ProcessDiagramErrorModal/__mocks__/ProcessDiagramErrorModal.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+const MockedProcessDetailsErrorModal = () => {
```
Comment (AjayJagan) [inline ui-packages/packages/process-details/src/envelope/components/ProcessDiagramErrorModal/__mocks__/ProcessDiagramErrorModal.tsx]: should it be MockedProcessDiagramErrorModal?

**Review by pefernan — APPROVED** [review]
+1 thanks!

**Pull request merged**